package.json

```diff
@@ -28,11 +29,15 @@
   "main": "index.js",
   "repository": {
     "type": "git",
-    "url": "https://github.com/mozilla/universal-search-addon"
+    "url": "git+https://github.com/mozilla/universal-search-addon.git"
   },
   "scripts": {
     "build": "gulp build",
     "lint": "gulp lint",
+    "release": "npm run release-patch",
+    "release-major": "npm version major && npm run build",
+    "release-minor": "npm version minor && npm run build",
+    "release-patch": "npm version patch && npm run build",
```

Comment:
We talked about it earlier, but I added the `release` scripts to this PR instead of doing a separate one.